independent-projects/tools/devtools-common/src/main/java/io/quarkus/registry/model/Release.java

```diff
@@ -15,6 +16,9 @@
     @JsonProperty("quarkus-core")
     String getQuarkusCore();
```

Comment:
I think we should get rid of this property and stick with your proposed property only

Reply:
It will surely keep evolving. I wouldn't remove it though now, knowing the Quarkus version the extension was built with could be pretty important. It's not exactly the same as "compatible with".